entrypoint/versionbump/cilium_test.go

```diff
@@ -44,7 +44,7 @@ var _ = Describe("VersionTemplate Upgrade:", func() {
 					},
 				},
 			},
-			InstallUpgrade: customflag.ServiceFlag.InstallUpgrade,
+			InstallUpgrade: customflag.ServiceFlag.InstallMode.String(),
```

Comment:
Shouldn't it be `InstallMode` instead of `InstallUpgrade`?

Reply:
It's not installmode?

Reply:
not the Flag but the var name